src/compositemark/errorband.ts

```diff
@@ -57,9 +57,9 @@ export interface ErrorBandConfig extends ErrorBandPartsMixins {
    * The line interpolation method for the error band. One of the following:
    * - `"linear"`: piecewise linear segments, as in a polyline.
    * - `"linear-closed"`: close the linear segments to form a polygon.
-   * - `"step"`: alternate between horizontal and vertical segments, as in a step function.
-   * - `"step-before"`: alternate between vertical and horizontal segments, as in a step function.
-   * - `"step-after"`: alternate between horizontal and vertical segments, as in a step function.
+   * - `"step"`: Produces a piecewise constant function (a step function) consisting of alternating horizontal and vertical lines. The y-value changes at the midpoint of each pair of adjacent x-values.
```

Comment:
small p? (word after colon should not be capitalized.  -- same for other lines.

Reply:
It's odd to say "produces" for only a few but not all. We should remove it.

Reply:
@kanitw Now you can review this. I have removed "produces" as suggested.